sidm/definitions/hists.py

```diff
@@ -969,28 +969,286 @@ def obj_eta_phi(obj, nbins_x=None, xmin=None, xmax=None, nbins_y=None, ymin=None
         evt_mask=lambda objs: ak.num(objs["ljs"]) > 1,
     ),
     # matchedjet
+    # matchedjet
```

Comment:
extra comment (not a big deal, but I'll leave this comment so we remember to remove it if we wind up making other changes anyway)